docs/apache-airflow-providers-elasticsearch/index.rst

```diff
@@ -56,7 +56,7 @@ Package apache-airflow-providers-elasticsearch
 `Elasticsearch <https://https//www.elastic.co/elasticsearch>`__
 
 
-Release: 1.0.2
+Release: 1.0.3
```

Comment:
Shall the [Changelog](https://github.com/apache/airflow/blob/8a3fe4b3875cebd457022048f271346bb45b7ac7/docs/apache-airflow-providers-elasticsearch/index.rst#changelog) section of this file also be updated?

Reply:
Oh yeah, thanks updated